lib/liquid/standardfilters.rb

```diff
@@ -317,6 +317,16 @@ def default(input, default_value = "".freeze)
 
     private
 
+    def to_integer(num)
+      return num if num.is_a?(Integer)
+      num = num.to_s
+      begin
+        Integer(num)
+      rescue ::ArgumentError
+        raise Liquid::ArgumentError, "invalid integer '#{num}'"
```

Comment:
do we really need to put the `#{num}` in here? does that give us much? I'm concerned now about the possible XSS you mentioned.